### ISSUE TITLE
dynamically_import_fxn breaking for stewi

### ISSUE THREAD

**Issue by bl-young** [USEPA/flowsa#83]
Accessing the FBS_outside_flowsa functions through the dynamic import is failing for stewi related data. I think the yaml's need to be revised

```ModuleNotFoundError: No module named 'flowsa.data_source_scripts.stewicombo'```

**Comment by bl-young** [comment]
@catherinebirney  sorry  I didn't catch this on your last pull request - only tested the FBAs.

**Comment by catherinebirney** [comment]
I'm having issues running CAP_HAP_national_2017 to help address this issue. 

`File "...flowsa\data_source_scripts\stewiFBS.py", line 40, in stewicombo_to_sector`
    `import stewicombo`
`ModuleNotFoundError: No module named 'stewicombo'`

I've had this error before - and did fix it at some point. It might be because stewi is not in the package requirements/setup? For fedelemflowlist and esupy, we define those module names in setup

**Comment by bl-young** [comment]
its looking for a .py file within flowsa called stewicombo.py
this https://github.com/USEPA/flowsa/commit/2c634682734e416245ce97fafae9dac1e837793c fixes it I think. need to confirm and I will open a pull request

**Comment by bl-young** [comment]
sorry read too quickly, stewicombo should be installed when you install the whole stewi package.

**Comment by catherinebirney** [comment]
yep - `pip install -e standardizedinventories` fixed my issue. The CAP_HAP fbs is running for me, so I think the changes to the source name worked
